dragula.js

```diff
@@ -155,11 +161,11 @@ function dragula (initialContainers, options) {
     if (drake.dragging && _mirror) {
       return;
     }
-    if (isContainer(item)) {
+    var handle = item;
+    if (isContainer(item, handle) && !o.allowNestedContainers) {
       return; // don't drag container itself
     }
-    var handle = item;
-    while (getParent(item) && isContainer(getParent(item)) === false) {
+    while (getParent(item) && isContainer(getParent(item), handle) === false) {
```

Comment:
`getParent(item)` should be cached as it executes the function twice (I know I'm getting into the original code, but it lacks performance tweaks), without any apparent reason.